token-metadata/program/src/processor/burn/nonfungible_edition.rs

```diff
@@ -124,36 +148,62 @@ pub(crate) fn burn_nonfungible_edition(ctx: &Context<Burn>) -> ProgramResult {
     };
     spl_token_close(params)?;
 
-    close_program_account(
-        ctx.accounts.metadata_info,
-        ctx.accounts.authority_info,
-        Key::MetadataV1,
-    )?;
-    close_program_account(edition_info, ctx.accounts.authority_info, Key::EditionV1)?;
-
     //       **EDITION HOUSEKEEPING**
     // Set the particular bit for this edition to 0 to allow reprinting,
     // IF the print edition owner is also the master edition owner.
     // Otherwise leave the bit set to 1 to disallow reprinting.
```

Comment:
Yeah, a creator should be able to clean up their mistake but a holder burning shouldn't enable a remint because it's assumed a creator wouldn't transfer a mistake, I guess.